public/api/conf/1.0/schemas/subSchemas/ultimateParent.json

```diff
@@ -12,19 +12,19 @@
       "type": "boolean"
     },
     "companyName": {
-      "description": "Name of the ultimate parent company",
+      "description": "Name of the ultimate parent company.",
       "$ref": "../definitions/definitions.json#/definitions/globalCompanyNameType"
     },
     "ctutr": {
-      "description": "Ultimate parent's Corporation Tax Unique Taxpayer Reference",
+      "description": "Ultimate parent's Corporation Tax Unique Taxpayer Reference.",
       "$ref": "../definitions/definitions.json#/definitions/utrType"
     },
     "sautr": {
       "description": "Ultimate parent company's self assessment Unique Taxpayer Reference.",
       "$ref": "../definitions/definitions.json#/definitions/utrType"
     },
     "countryOfIncorporation": {
-      "description": "Country of incorporation where the ultimate parent company is not a UK company. A valid ISO 3166-1 2 Alpha Country Code",
+      "description": "Country of incorporation where the ultimate parent company is not a UK company. A valid ISO 3166-1 2 Alpha Country Code.",
```

Comment:
Wasn't one of the things @burns238 mentioned on Friday to not have punctuation at the end as it looks weird on the dev hub?